up42/asset.py

```diff
@@ -103,14 +103,24 @@ def update_metadata(
         Update the metadata of the asset.
 
         Args:
-            title: The title string to be assigned to the asset.
-            tags: A list of tag strings to be assigned to the asset.
+            title: The title string to be assigned to the asset. No value will keep the existing title.
+            tags: A list of tag strings to be assigned to the asset. No value will keep the existing tags.
 
         Returns:
             The updated asset metadata information
         """
         url = endpoint(f"/v2/assets/{self.asset_id}/metadata")
-        body_update = {"title": title, "tags": tags, **kwargs}
+        if kwargs:
+            logger.info(
+                f"{','.join([key for key in kwargs.keys()])} values are not allowed to update in asset metadata."
```

Comment:
if we are not allowing any key from kwargs, should we not just drop the `kwargs` argument altogether?

Reply:
no change in method signature, otherwise we break user code.

Reply:
![Screenshot 2024-03-13 at 14 17 57](https://github.com/up42/up42-py/assets/3291933/078e127e-c948-4da6-b86c-8913bb9f33f3)

Reply:
Please just deprecate it, this block is not needed